implementations/micrometer-registry-graphite/src/main/java/io/micrometer/graphite/GraphiteConfig.java

```diff
@@ -49,12 +49,13 @@ default String prefix() {
 
     /**
      * @return Whether Graphite tags should be used, as opposed to a hierarchical naming convention.
-     * Defaults to {@code true}.
+     * Defaults to true if a value is present for {@link GraphiteConfig#tagsAsPrefix}.
```

Comment:
Don't we want this to be `true` if a value is NOT present for `tagsAsPrefix`? This enables dimensional tags, `tagsAsPrefix` is for hierarchical naming. /cc @snicoll

Reply:
Yes, we want the opposite. 

The way I understood this is that if `tagsAsPrefix` has a value, you'd want the "non-native" way of tagging. If you are upgrading to a new version of micrometer that provides a native version of tagging, I think you should not be affected by this change.

So, a user upgrading:

* If they have `tagsAsPrefix` set we should use that (native tagging not enabled since it's going to ignore those anyway)
* If they don't have `tagsAsPrefix` set, then offer the native tagging (native tagging enabled since the user didn't express a use of the previous mechanism).

Makes sense? I think that's a fair summary of our discussion so far.

Reply:
Sounds right to me. I wanted to make sure I wasn't mistaken.

Reply:
Yeah, I'm running on very little sleep with quarantine toddler.

Updated in 03ea12adfc7666beb4b8e38e593d3d43c1473781

Reply:
No problem. Thanks for updating and take care.